src/styles/main.scss

```diff
@@ -1,3 +1,57 @@
+$cell-size: 100px;
+$cell-gap: 1px;
+$cell-shift: $cell-size + $cell-gap;
+$calendar-paddings: 10px;
+$calendar-width: ($cell-shift * 7) + ($cell-gap * 6) + ($calendar-paddings * 2);
+$days: mon 1, tue 2, wed 3, thu 4, fri 5, sat 6, sun 7;
+
 body {
   margin: 0;
 }
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  padding: $calendar-paddings;
+  width: $calendar-width;
+  gap: $cell-gap;
+  list-style-type: none;
+
+  @for $day from 1 through 31 {
+    & > :nth-child(#{$day})::before {
+      content: "#{$day}";
+    }
+  }
+
+  &__day {
+    box-sizing: border-box;
```

Comment:
Let's set this property using `*` selector, as it will be applied as the general rule for all elements in your project